### PR TITLE
Update Links and branding for .NET Core

### PR DESCRIPTION
**Pull request by BillWagner**
Page destinations moved.
A couple branding changes for RTM.

Please review @richlander @mairaw

### CODE REVIEW

File: projects/NETCore.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
 .NET Core has two major components. It includes a small runtime that is built from the same codebase as the .NET Framework CLR. The .NET Core runtime includes the same GC and JIT (RyuJIT), but doesn’t include features like Application Domains or Code Access Security. The runtime is delivered on NuGet, via the Microsoft.CoreCLR package.
 
 .NET Core also includes the base class libraries. These libraries are largely the same code as the .NET Framework class libraries, but have been factored (removal of dependencies) to enable us to ship a smaller set of libraries. These libraries are shipped as System.* NuGet packages on NuGet.org.
 
 ## Project Details
 
-* [Project Info Site](https://dotnet.github.io/core)
+* [Project Info Site](https://dotnet.github.io)
```
Comment (mairaw) [inline projects/NETCore.md]: should this link to https://www.microsoft.com/net/core instead?

File: projects/NETCore.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 * Project Code Repos:  [.NET Core Framework](https://github.com/dotnet/corefx), [.NET Core Common Language Runtime]( https://github.com/dotnet/coreCLR)
```
Comment (mairaw) [inline projects/NETCore.md]: @blackdwarf should we add CLI repo to this list? anything else?

**Comment by mairaw** [comment]
LGTM modulo some questions. Thanks @BillWagner!

File: projects/NETCore.md (hunk 2 above)
Comment (BethMassi) [inline projects/NETCore.md]: Shouldn't we be pointing to https://www.microsoft.com/net/core/platform (we're updating this page now) or https://docs.microsoft.com/dotnet/ when docs go live on Monday?

Comment (BillWagner) [inline projects/NETCore.md]: I can add the first if you'd like. 

Should the doc link go to the 'core' or the .NET home page? 

Also, @BethMassi can you review the text at line 5? I want to make sure that is still accurate in terms of how it positions core in contrast to the full framework.

File: projects/NETCore.md [hunk 3]
```diff
@@ -17,6 +17,6 @@
 * [Contribute](https://github.com/dotnet/corefx/blob/master/CONTRIBUTING.md)
 * [Discussions](http://forums.dotnetfoundation.org/)
 * [Samples](https://github.com/Microsoft/dotnetsamples)
-* [Blog](http://blogs.msdn.com/b/dotnet/)
+* [Blog](http://blogs.msdn.microsoft.com/dotnet/)
 * Twitter: [@DotNet](https://twitter.com/dotnet)
-* [[WIP] Documentation](http://dotnet.readthedocs.org/)
+* [Documentation](http://docs.microsoft.com/dotnet/core/index)
```
Comment (mairaw) [inline projects/NETCore.md]: also wondering if we want to link to core directly as in here or to https://docs.microsoft.com/dotnet/

File: projects/NETCore.md (hunk 2 above)
Comment (blackdwarf) [inline projects/NETCore.md]: @mairaw yup, please add the cli repo on it. I think with coreclr, cli and corefx we are fine.

**Comment by BillWagner** [comment]
Responded to feedback. @BethMassi @richlander Is this ready to go?

File: projects/NETCore.md [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
 
 .NET Core has two major components. It includes a small runtime that is built from the same codebase as the .NET Framework CLR. The .NET Core runtime includes the same GC and JIT (RyuJIT), but doesn’t include features like Application Domains or Code Access Security. The runtime is delivered on NuGet, via the Microsoft.CoreCLR package.
 
 .NET Core also includes the base class libraries. These libraries are largely the same code as the .NET Framework class libraries, but have been factored (removal of dependencies) to enable us to ship a smaller set of libraries. These libraries are shipped as System.* NuGet packages on NuGet.org.
 
 ## Project Details
 
-* [Project Info Site](https://dotnet.github.io/core)
-* Project Code Repos:  [.NET Core Framework](https://github.com/dotnet/corefx), [.NET Core Common Language Runtime]( https://github.com/dotnet/coreCLR)
+* [Project Info Site](https://dotnet.github.io)
```
Comment (mairaw) [inline projects/NETCore.md]: this should be changed to some other site. Where are we redirecting dotnet.github.io to @blackdwarf @bradygaster?

**Comment by mairaw** [comment]
I had one question pending and one more comment now.

**Comment by BillWagner** [comment]
@mairaw It should be http://docs.microsoft.com/dotnet Update coming

File: projects/NETCore.md (hunk 4 above)
Comment (richlander) [inline projects/NETCore.md]: How about: https://github.com/dotnet/core

File: projects/NETCore.md [hunk 5]
```diff
@@ -1,22 +1,22 @@
 # .NET Core
 
-.NET Core is the small optimized runtime that is the basis of [ASP.NET Core 5](http://www.dotnetfoundation.org/aspnet-5). It currently runs on Windows, and will be extended to support Linux and Mac. It is a high-performance and modular design, and supports full side by side to make it easy to adopt new .NET Core versions without affecting other apps. These products are actively developed by the .NET team and in collaboration with a community of open source developers. Together we are dedicated to improving and extending the .NET platform with new features and for new scenarios.
+.NET Core is the small optimized runtime that is the basis of [ASP.NET Core](http://www.dotnetfoundation.org/aspnet-core). It currently runs on Linux, Mac, and Windows. It is a high-performance and modular design, and supports full side by side to make it easy to adopt new .NET Core versions without affecting other apps. These products are actively developed by the .NET team and in collaboration with a community of open source developers. Together we are dedicated to improving and extending the .NET platform with new features and for new scenarios.
```
Comment (richlander) [inline projects/NETCore.md]: This text is now a bit wrong. It should say:

- .NET Core is blah.
- ASP.NET Core builds on top of it.

File: projects/NETCore.md (hunk 3 above)
Comment (richlander) [inline projects/NETCore.md]: Probably the more general one since the URLs are still changing.

**Comment by mairaw** [comment]
LGTM. You have one outstanding comment from Rich though.

File: projects/NETCore.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
...
+.NET Core is a high-performance and modular design, and supports full side by side to make it easy to adopt new .NET Core versions without affecting other apps. It currently runs on Linux, Mac, and Windows. [ASP.NET Core](http://www.dotnetfoundation.org/aspnet-core) is built on top of .NET Core. ASP.NET Core is a high-performance and modular design, and supports full side by side to make it easy to adopt new versions without affecting other apps. These products are actively developed by the .NET team and in collaboration with a community of open source developers. Together we are dedicated to improving and extending the .NET platform with new features and for new scenarios.
```
Comment (svick) [inline projects/NETCore.md]: > .NET Core is a high-performance and modular design

**has** a high-performance and modular design?

**Comment by BillWagner** [comment]
Thanks for the catch @mairaw Updated now.

@richlander Can you review the opening paragraph?

**Comment by richlander** [comment]
LGTM

**Pull request merged**